classes/submission/form/PKPSubmissionSubmitStep1Form.inc.php

```diff
@@ -163,7 +163,7 @@ function setSubmissionData($submission) {
 	/**
 	 * Add or update comments to editor
 	 */
-	function setCommentsToEditor($submissionId, $commentsToEditor, $userId, $query) {
+	function setCommentsToEditor($submissionId, $commentsToEditor, $userId, $query = null) {
```

Comment:
ah, I see it is null per default here, so forget the comment from the commit before... :-)